src/components/common/TodoCard/TodoCard.js

```diff
@@ -126,11 +148,79 @@ class TodoCard extends React.Component {
     return (
       <Image
         onClick={this.handlePausePlayClick}
-        icon={this.props.job.paused ? Image.ICONS.PAUSE : Image.ICONS.PLAY}
+        icon={this.getIconType(this.props.job)}
       />
     )
   }
 
+  oneSecondInterv () {
+    const { job } = this.props
+    const pausedFor = get(job, "pausedFor") * 1000
+
+    if (job.paused && job.state === JOB_STATE_STARTED) {
+      const now = Date.now()
+      const pausedAt = get(job, "pausedAt").getTime()
```

Comment:
this usage of get function is redundunt
just use plain dot notation job.property